gomock/callset.go

```diff
@@ -111,3 +131,19 @@ func (cs callSet) Failures() []*Call {
 	}
 	return failures
 }
+
+// AllExpectedCallsSatisfied returns true in case all expected calls in this callSet are satisfied.
+func (cs callSet) AllExpectedCallsSatisfied() bool {
```

Comment:
re: naming, this is a mouthful, can we call this `Satisfied` instead?
```suggestion
func (cs callSet) Satisfied() bool {
```

Reply:
👍